ext/project_data.yaml

```diff
@@ -35,7 +35,6 @@ gem_platform_dependencies:
       win32-dir: '~> 0.4.9'
       win32-eventlog: '~> 0.6.2'
       win32-process: '~> 0.7.4'
-      win32-security: '~> 0.2.5'
```

Comment:
Unfortunately we can't remove this here just yet... `project_data.yaml` is used for a couple of things:

* Development workflows (including running specs in CI), when performing a `bundle install` pick up these gem dependencies.  For the most part, this is unlikely to have many negative consequences (but it's possible there's some scenario that could feel some fallout)
* During the release process when we cut gems.  This is the bit we have to be more careful about since excluding the gem here is roughly equivalent to cutting it out of the `puppet-win32-ruby` repo for the sake of the MSI packaging.  For this reason, we should wait for a deprecation cycle / next semver bump before we completely yank this (and it should be synced up with the removal from the vendored gems).